src/action/auth-mobile.js

```diff
@@ -164,18 +175,38 @@ class AuthAction {
     await this._wallet.checkPassword();
   }
 
-  _getFromKeyStore(key) {
+  async _getFromKeyStore(key) {
+    const vKey = `${VERSION}_${key}`;
+    const credentials = await this._Keychain.getInternetCredentials(vKey);
+    if (credentials) {
+      return credentials.password;
+    } else {
+      return this._migrateKeyStoreValue(key); // TODO: remove from future version
```

Comment:
Could add a backlog ticket for this todo

Reply:
It's already removed in https://github.com/lightninglabs/lightning-app/tree/mobile/bare-react-native/mobile-bare